.github/workflows/changelog-flow.yml

```
@@ -0,0 +1,34 @@
+name: Generate Changelog
+on:
+  release:
+    types: [published]
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: master
```

```suggestion
          ref: main
```
Should this be main?